.github/workflows/Documenter.yml

```diff
@@ -1,20 +1,39 @@
 name: Documenter
 on:
   push:
-    branches: [main]
-    tags: [v*]
+    branches:
+      - main
+    tags: '*'
   pull_request:
-    branches: [main]
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
 jobs:
-  Documenter:
-    name: Documentation
-    runs-on: ubuntu-latest
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      DISPLAY: ':0'
     steps:
       - uses: actions/checkout@v2
-      - uses: julia-actions/julia-buildpkg@latest
-      - uses: julia-actions/julia-docdeploy@latest
+      - uses: julia-actions/setup-julia@v1
+      - uses: julia-actions/cache@v1
+        with:
+          cache-registries: "true"
+      - name: Install documentation dependencies
+        run: sudo apt-get update && sudo apt-get install -y xorg-dev mesa-utils xvfb libgl1 freeglut3-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev libxext-dev
+      - name: Install pkgs dependencies
+        run: xvfb-run -s '-screen 0 1024x768x24' julia --project=docs -e 'using Pkg; Pkg.instantiate(); Pkg.precompile()'
```

Comment:
It doesn't look like GLMakie is used anywhere?  If it isn't, then no need for the xvfb stuff.

Reply:
This for docs, later it will be used for plotting spheres. I was planning in also adding recipes for that.

Reply:
Smallish (720x360) spheres should be fine with CairoMakie ;)